src/components/controls/TransitToggleButtonBar.tsx

```diff
@@ -25,28 +26,33 @@ class TransitToggleButtonBar extends React.Component<ITtransitToggleButtonBarPro
                     toggleActivity={this.toggleType}
                     toggled={this.props.selectedTransitTypes.includes(TransitType.BUS)}
                     type={TransitType.BUS}
+                    disabled={this.props.disabled}
                 />
                 <TransitToggleButton
                     toggleActivity={this.toggleType}
                     toggled={this.props.selectedTransitTypes.includes(TransitType.TRAM)}
                     type={TransitType.TRAM}
+                    disabled={this.props.disabled}
                 />
                 <TransitToggleButton
                     toggleActivity={this.toggleType}
                     toggled={this.props.selectedTransitTypes.includes(TransitType.TRAIN)}
                     type={TransitType.TRAIN}
+                    disabled={this.props.disabled}
                 />
                 <TransitToggleButton
                     toggleActivity={this.toggleType}
                     toggled={this.props.selectedTransitTypes.includes(TransitType.SUBWAY)}
                     type={TransitType.SUBWAY}
+                    disabled={this.props.disabled}
                 />
                 <TransitToggleButton
                     toggleActivity={this.toggleType}
                     toggled={this.props.selectedTransitTypes.includes(TransitType.FERRY)}
                     type={TransitType.FERRY}
+                    disabled={this.props.disabled}
                 />
-                { this.props.disabled &&
+                { this.props.blurred &&
```

Comment:
If blurred puts on s.disabledOverlay --> rename as s.blurredOverlay

Reply:
Why would the parents need to know it is an overlay?

Reply:
missunderstood your comment, this is now fixed.